--- Makefile~	Mon Aug  9 05:04:14 1999
+++ Makefile	Fri Sep  3 22:16:34 1999
@@ -4,7 +4,8 @@
 INCDIR  = -I/usr/X11R6/include
 LIBDIR  = -L/usr/X11R6/lib
 LIBS    = -lXpm -lXext
-CFLAGS  = $(INCDIR) -g -DICONDIR=\"$(ICONDIR)\" -O2 -Wall
+OPTS	= -O2 -Wall
+CFLAGS  = $(INCDIR) -g -DICONDIR=\"$(ICONDIR)\" $(OPTS)
 
 wmbattery: wmbattery.c wmbattery.h
 	$(CC) $(CFLAGS) $(LIBDIR) $(LIBS) wmbattery.c -o wmbattery
